psa-crypto-sys/build.rs

```diff
@@ -209,22 +276,17 @@ mod common {
 #[cfg(all(feature = "interface", not(feature = "operations")))]
 mod interface {
     use super::common;
-    use std::env;
-    use std::io::{Error, ErrorKind, Result};
+    use std::io::Result;
 
     // Build script when the interface feature is on and not the operations one
     pub fn script_interface() -> Result<()> {
-        if let Ok(include_dir) = env::var("MBEDTLS_INCLUDE_DIR") {
-            common::configure_mbed_crypto()?;
-            common::generate_mbed_crypto_bindings(include_dir.clone(), false)?;
-            let _ = common::compile_shim_library(include_dir, true, false)?;
-            Ok(())
-        } else {
-            Err(Error::new(
-                ErrorKind::Other,
-                "interface feature necessitates MBEDTLS_INCLUDE_DIR environment variable",
-            ))
-        }
+        let include_dir = common::get_external_mbedtls_include_only().unwrap();
+
+        // TODO: Does interface need the vendored mbedtls?
```

Comment:
@gowthamsk-arm is this needed? I don't think it's needed.

Reply:
I don't think it's needed. It was added in this [patch](https://github.com/parallaxsecond/rust-psa-crypto/commit/c66aa62349de40ceb395c681f0f8abe126916707). 

@egrimley-arm, just to confirm, was there a particular reason (that we are missing) to configure the vendored medtls even for the interface feature?

Reply:
I dropped this in a separate patch just on the off chance it needs to be reverted.

Reply:
Digging into this more, this is not what you wanted as the interface feature was partially broken as well. I kept the behavior the same until now, as I noticed that it was borked. See the commit message.